packages/modeling/src/operations/hulls/hullPoints2.js

```diff
@@ -25,11 +25,9 @@ export const hullPoints2 = (uniquePoints) => {
   })
 
   // sort by polar coordinates
-  points.sort((pt1, pt2) => pt1.angle < pt2.angle
-    ? -1
-    : pt1.angle > pt2.angle
-      ? 1
-      : pt1.distSq < pt2.distSq ? -1 : pt1.distSq > pt2.distSq ? 1 : 0)
+  points.sort((pt1, pt2) => pt1.angle !== pt2.angle
```

Comment:
I don't remember this change... do you?

Reply:
Yea that was me in the linting commit e1e20d1